docs/user/server-side-search/index.rst

```diff
@@ -6,6 +6,9 @@ this is powered by Elasticsearch_.
 You can search all projects at https://readthedocs.org/search/,
 or search only on your project from the :guilabel:`Search` tab of your project.
```

Comment:
I think we should remove this paragraph as well. Results from that link look _terrible_ and we have killed that page on the new dashboard.

It probably makes sense here to mention how to use the search currently. This is by opening the flyout and clicking on the "Search" input there. That should be enough. If they aren't using addons they will get the old behavior and if they are using the addons, they will get the new nice modal.

We can also probably add an image of the new search addons modal if we think it adds value here.